cmd/drone-server/setup.go

```diff
@@ -117,11 +117,23 @@ func setupGogs(c *cli.Context) (remote.Remote, error) {
 
 // helper function to setup the Gitea remote from the CLI arguments.
 func setupGitea(c *cli.Context) (remote.Remote, error) {
-	return gitea.New(gitea.Opts{
+	if !c.IsSet("gitea-client") {
```

Comment:
I believe it is better to use early returns on a non negated condition. Eases understanding. But this is no biggie.